scipy/stats/_continuous_distns.py

```diff
@@ -2900,6 +2900,135 @@ def _entropy(self, c):
 genhalflogistic = genhalflogistic_gen(a=0.0, name='genhalflogistic')
 
 
+class genhyperbolic_gen(rv_continuous):
+    r"""A generalized hyperbolic continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `genhyperbolic` is:
+
+    .. math::
+
+        f(x, \lambda, \alpha, \beta, \delta, \mu) =
+           \frac{(\gamma/\delta)^\lambda}{\sqrt{2\pi}K_\lambda(\delta \gamma)}
+           e^{\beta (x - \mu)} \times \frac{K_{\lambda - 1/2}
+           (\alpha \sqrt{\delta^2 + (x - \mu)^2})}
+           {(\sqrt{\delta^2 + (x - \mu)^2} / \alpha)^{1/2 - \lambda}}
+
+    for :math:`x \in (- \infty; \infty)`,
+    :math:`\lambda, \mu \in \mathbb{R}`,
+    :math: `0 \le |\beta| < \alpha ` \beta,
```

Comment:
The closing ` seems to be in the wrong place. That might be causing doc build failure.
```
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/stats/__init__.py:docstring of scipy.stats.genhyperbolic:55: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/stats/__init__.py:docstring of scipy.stats.genhyperbolic:59: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

Reply:
Havent looked at that in much detail yet, but I have managed to build the html documentation locally

Reply:
Great! That will help.

I was saying that
:math: `0 \le |\beta| < \alpha ` **\beta,**
Is left outside of the backtick, and that might be causing a problem.